application/app/views/shared/_file_stats_widget.html.erb

```diff
@@ -0,0 +1,70 @@
+<%# Badge counts (ensure values default to 0) %>
+<% completed ||= 0 %>
+<% pending ||= 0 %>
+<% cancelled ||= 0 %>
+<% error ||= 0 %>
+<% total ||= 0 %>
+
+<%
+  if total == 0
+    percent = 0
+  else
+    percent = ((completed + cancelled + error) * 100 / total.to_f).round(1).to_i
+  end
+%>
+<% offset = (114 - (114 * percent.to_f / 100)).round(1).to_i %>
+
+<%# Default colors if not passed %>
+<% progress_color ||= "#0d6efd" %>
+<% background_color ||= "#e9ecef" %>
+<% text_color ||= "#212529" %>
```

Comment:
Is there any way of reading those colors from SCSS files instead of hardcoding this in the view template?

Reply:
No that I know of. The colors are used directly in the SVG definition for the circle and internal text